config/config.yml

```diff
@@ -259,6 +259,22 @@ FREEFORM_CATEGORY_NAME: 'Additional Tags'
 COMMON_CATEGORY_NAME: 'Common'
 BANNED_CATEGORY_NAME: 'Banned'
 
+# Wrangler is considered inactive if they haven't wrangled for this many days
+WRANGLING_INACTIVITY_THRESHOLD: 14
+# Wrangler is considered for hiatus if they haven't wrangled for this many total days
+WRANGLING_HIATUS_THRESHOLD: 21
+WRANGLING_HIATUS_FORM_URL: "https://example.com/hiatus_form_placeholder"
+ASSIGNED_FANDOMS_POLICY_URL: "https://example.com/Assigned_Fandoms_Policy_placeholder"
+USERS_EXCLUDED_FROM_WRANGLING_INACTIVITY:
+- AO3_Wrangling_Project
+- zz9pzza
+- Qem_Chibati
+- AO3_Support
+- AO3_Tag_Wranglers
+- testy_wrangler
+- elz
+- astolat
```

Comment:
I talked to TW and we have some changes here! Please remove the last two names and add TWSupervisor_Fandom_Hold. I'll update the list on the issue. 

(Although now that I think about it, @brianjaustin and/or @zz9pzza, should we leave this blank and just have Systems add the names in when this is deployed? These names won't be relevant to other instances.)

Reply:
Yes, let's configure this via Ansible instead of here